Plugins/NativeCamera/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+set(SOURCES 
+    "Include/Babylon/Plugins/NativeCamera.h"
+    "Source/NativeCamera.cpp"
+    "Source/NativeCamera.h"
+    "Source/NativeVideo.cpp"
+    "Source/NativeVideo.h")
+
+if(APPLE)
+    set(SOURCES ${SOURCES} "Source/NativeCamera${BABYLON_NATIVE_PLATFORM}.mm")
+else()
+    set(SOURCES ${SOURCES} "Source/NativeCamera${BABYLON_NATIVE_PLATFORM}.cpp")
+endif()
+
+if (APPLE OR ANDROID OR (WIN32 AND NOT WINDOWS_STORE))
```

Comment:
nit: Not even really a nit, just an interesting option I didn't know about. Apparently CMake has a [`return`](https://cmake.org/cmake/help/v3.12/command/return.html) command, in case you wanted to early-out instead of encapsulating all the functionality in a condition.